crates/assistant2/src/active_thread.rs

```diff
@@ -35,6 +34,7 @@ pub struct ActiveThread {
     editing_message: Option<(MessageId, EditMessageState)>,
     expanded_tool_uses: HashMap<LanguageModelToolUseId, bool>,
     last_error: Option<ThreadError>,
+    lua_language: Option<Arc<Language>>, // Used for syntax highlighting in the Lua script tool
```

Comment:
Holding on to Lua and initializing it in the Thread seems a bit odd.

If we know we need it, should we just load it in `languages` with all the rest of them?